src/coreComponents/constitutive/contact/CoulombFriction.hpp

```diff
@@ -322,6 +333,22 @@ inline void CoulombFrictionUpdates::updateFractureState( localIndex const k,
     real64 const yield = tauNorm - limitTau;
 
     fractureState = yield < 0 ? FractureState::Stick : FractureState::Slip;
+
+    real64 const slip[2] = { dispJump[1] - oldDispJump[1],
+                             dispJump[2] - oldDispJump[2] };
+    if( yield < 0 )
+    {
+      // The slip is only elastic: we add the full slip to the elastic one
+      LvArray::tensorOps::add< 2 >( m_elasticSlip[k], slip );
+    }
+    else
+    {
+      m_plasticSlip[k][0] += tractionVector[1] / m_shearStiffness;
+      m_plasticSlip[k][1] += tractionVector[2] / m_shearStiffness;
```

Comment:
I think that this was wrong and this is the elastic slip. 

![Screenshot 2024-11-21 at 17 28 19](https://github.com/user-attachments/assets/a3a8ac8f-f3d0-4ca3-9426-b2e1815932f1)

Reply:
and we only need to keep track of one of the two.

Reply:
done